nodes.erl

```diff
@@ -73,7 +78,7 @@ query_to_commit(OtherNodes) ->
     broadcast(OtherNodes, {query, self()}).
 
 broadcast(Nodes, Message) ->
-    lists:map(fun(Node) -> Node ! Message end, Nodes).
+    [Node ! Message || Node <- Nodes].
```

Comment:
Yes especially with OTP Elixir made it really easy to get started without having to write giant nested tuples for configuration